tldr/change_impact.py

```
@@ -221,12 +242,20 @@ def collect_test_files(node: dict):
                 for caller in node.get("callers", []):
                     collect_test_files(caller)
 
-            collect_test_files(impact.get("callers", {}))
+            for target_node in impact.get("targets", {}).values():
+                collect_test_files(target_node)
 
         except Exception:
             # If impact analysis fails for a function, continue
             pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)